README.md

```diff
@@ -150,47 +184,35 @@ docker run -p 30333:30333 -p 9933:9933 -p 9944:9944 \
 docker run -p 30334:30333 -p 9934:9933 -p 9945:9944 \
            -it paritytech/millau-bridge-node --dev --tmp \
            --rpc-cors=all --unsafe-rpc-external --unsafe-ws-external
-```
-
-Notice that the `docker run` command will accept all the normal Substrate flags. For local
-development you should at minimum run with the `--dev` flag or else no blocks will be produced.
 
-Then we need to initialize and run the relayer:
-
-```bash
 docker run --network=host -it \
-        paritytech/substrate-relay init-bridge RialtoToMillau \
+        paritytech/substrate-relay init-bridge MillauToRialto \
         --target-host localhost \
-        --target-port 9945 \
+        --target-port 9944 \
         --source-host localhost \
-        --source-port 9944 \
+        --source-port 9945 \
         --target-signer //Alice
 
 docker run --network=host -it \
-        paritytech/substrate-relay relay-headers RialtoToMillau \
+        paritytech/substrate-relay relay-headers MillauToRialto \
         --target-host localhost \
-        --target-port 9945 \
+        --target-port 9944 \
         --source-host localhost \
-        --source-port 9944 \
-        --target-signer //Bob \
-```
-
-You should now see the relayer submitting headers from the Millau chain to the Rialto chain.
+        --source-port 9945 \
+        --target-signer //Bob
 
-If you don't want to use the published Docker images you can build images yourself. You can do this
-by running the following commands at the top level of the repository.
-
-```bash
-# In `parity-bridges-common` folder
-docker build . -t local/rialto-bridge-node --build-arg PROJECT=rialto-bridge-node
-docker build . -t local/millau-bridge-node --build-arg PROJECT=millau-bridge-node
-docker build . -t local/substrate-relay --build-arg PROJECT=substrate-relay
+docker run --network=host -it \
+        paritytech/substrate-relay relay-messages MillauToRialto \
```

Comment:
Same thing here, to relay messages we need the other side of the bridge running (Rialto -> Millau)